mcp/src/apis/getUsers.ts

```diff
@@ -5,13 +5,22 @@ import { ApiFactory } from '../shared/boilerplate/src/types.js';
 const inputSchema = {
   includeTimezone: z
     .boolean()
+    .default(true)
     .describe(
-      'If true, includes the timezones for each user. Not needed for most cases.',
+      'If true, includes the timezones for each user. Defaults to true.',
     ),
+  user_id: z
+    .string()
+    .optional()
+    .describe('The Slack user id (e.g. U0736TW20) of a single user to retrieve.'),
+  email: z
+    .string()
+    .optional()
+    .describe('The email address of a single user to retrieve.'),
   keyword: z
     .string()
     .min(0)
-    .nullable()
+    .optional()
```

Comment:
We can't use `optional`, it breaks things. See https://github.com/timescale/tiger-slack-mcp-server/pull/13

Reply:
ugh. gross. ok